src/lib.rs

```diff
@@ -84,31 +84,36 @@ pub trait FileExt: AsRawFd {
     where
         N: AsRef<OsStr>,
     {
-        util::extract_noattr(sys::get_fd(self.as_raw_fd(), name.as_ref()))
+        // SAFETY: Implement I/O safety later.
```

Comment:
Note: this _should_ be safe as-is as we don't keep the file descriptor around. I.e., the API is unsafe, but the usage is safe.